lib/simplifyExpression/collectAndCombineSearch/divideLikeTerms.js

```diff
@@ -0,0 +1,178 @@
+const arithmeticSearch = require('../arithmeticSearch');
```

Comment:
Pretty much everything everything. It should do the same thing as multiply but with another operator. Should I just pass the operator "-" to multiplyLikeTerms? Will propably be cleaner